### DISCUSSION TITLE
where is the msa token stored after auth

### DISCUSSION THREAD

**Discussion by BrycensRanch** [PrismarineJS/mineflayer#2970] [Q&A]
please help 📦

**Comment by amoraschi** [comment]
If you didn't specify a folder in the createBot function, the credentials are cached im your `/.minecraft/nmp-cache/` folder by default.

**Reply by BrycensRanch** [reply]
wait, I'm sorry, my .minecraft folder? you mean ~/.minecraft/nmp-cache?
